### PR TITLE
ShlomiShitrit/issue18 - add custom messages 

### LINKED ISSUE #18: Add customable messages for operators, params etc

### PR DESCRIPTION
**Pull request by ShlomiShitrit**
Fix #18 

The feature is about giving the developer more flexibility with the bot messages.

The bot now have a prop named "_messages" which have  two props: "customMessages" and "slots".

The customMessages is used for customizing the messages in the main steps of the bot, like when the user chooses an attribute or an operator.

The slots is used for adding "intro" messages between the main steps.
there is no limit of the amount of messages in any slot.

There is an example in the demo.ts script.

### CODE REVIEW

**Comment by ShlomiShitrit** [comment]
> add custome messages for:
> 
> * "Enter 1 to continue"
> * Result messages
> 
> Also:
> 
> * Add as many intro messages as wanted
> * Option to change the display name of attribute

I think the option to change the display name of an attribute should be in another issue, I will open one now,

The rest tasks have been done.

Waiting for  your review @nadavWeisler

**Review by ShlomiShitrit — COMMENTED** [review]
Pull changes after merging PR of issue25 and resolved conflicts. The PR can be review now.

File: DBbot/classes/DBbot.ts
Comment (nadavWeisler) [inline DBbot/classes/DBbot.ts:28]: Remove comments

**Review by nadavWeisler — APPROVED** [review]

File: DBbot/classes/DBbot.ts
Comment (ShlomiShitrit) [reply DBbot/classes/DBbot.ts:28]: done

**Pull request merged**
